extensions/wikia/AdMixExperiment/scripts/AdMixExperiment.js

```diff
@@ -25,8 +27,14 @@ require(['ext.wikia.adEngine.adContext', 'wikia.abTest', 'wikia.throttle'], func
 		var visibleElementBeforeWrapperHeight;
 
 		// ad mix flags
-		var topRightAdFixed = !!(abTest.getGroup('AD_MIX') && abTest.getGroup('AD_MIX') !== 'AD_MIX_2');
-		var recircEnabled = !!(abTest.getGroup('AD_MIX') && abTest.getGroup('AD_MIX') !== 'CONTROL');
+		// AD_MIX_1: reloadFloatingMedrec === true && recircFixed === true
+		// AD_MIX_2: topRightAdFixed === true && recircFixed === true
+		// AD_MIX_3: reloadRecirc === true && recircFixed === true
+		// CONTROL: all === false
+		var reloadFloatingMedrec = !!(abTest.getGroup('AD_MIX') && abTest.getGroup('AD_MIX') === 'AD_MIX_1');
+		var topRightAdFixed = !!(abTest.getGroup('AD_MIX') && abTest.getGroup('AD_MIX') === 'AD_MIX_2');
+		var reloadRecirc = !!(abTest.getGroup('AD_MIX') && abTest.getGroup('AD_MIX') === 'AD_MIX_3');
+		var recircFixed = !!(abTest.getGroup('AD_MIX') && abTest.getGroup('AD_MIX') !== 'CONTROL');
```

Comment:
Lines 34-37
`var recircFixed = abTest.getGroup('AD_MIX') === 'CONTROL'` is sufficient